modules/base/rendering/renderablemodel.cpp

```diff
@@ -449,7 +540,7 @@ void RenderableModel::initializeGL() {
 
     _geometry->initialize();
     _geometry->calculateBoundingRadius();
-    setBoundingSphere(glm::sqrt(_geometry->boundingRadius()));
+    setBoundingSphere(glm::sqrt(_geometry->boundingRadius()) * _scaleVector.x);
```

Comment:
Wouldn't it be possible to specify different scale values for each direction?  In which case multiplying with 
`glm::compMax(_scaleVector)` (https://glm.g-truc.net/0.9.5/api/a00191.html) would give the better pessimistic bounding sphere